src/Package/Localization/Microsoft.Build.Localization.nuspec

```diff
@@ -7,7 +7,7 @@
     <authors>Microsoft</authors>
     <owners>Microsoft</owners>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
-    <licenseUrl>$licenseUrl$</licenseUrl>
+    <license type="expression">$licenseExpression$</license>
```

Comment:
BTW: `repository` metadata element is missing here. 

You can clean this up and make sure all metadata is set by using `$CommonMetadataElements$` variable like so:

https://github.com/dotnet/roslyn/blob/master/src/Dependencies/CodeAnalysis.Debugging/Microsoft.CodeAnalysis.Debugging.nuspec#L4

This variable is set by Arcade based on standard project properties here: https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/tools/Workarounds.targets#L121